golismero/api/net/web_utils.py

```diff
@@ -126,7 +126,8 @@ def data_from_http_response(response):
 
     # If we have no data, return None.
     if not response.data:
-        return None
+        from ..data.information.html import HTML
```

Comment:
Why this change? Shouldn't it be better to change the caller not to send None? If I understand this correctly we would be adding HTML elements to the database that are actually empty inside...